### PR TITLE
[Feature] GIF generation

### PR DESCRIPTION
**Pull request by Sqyphen**
**Description**
Adding README content describing a feature which would allow a series of images (each spanning a single day) to be turned into a animated collection (i.e. GIF)

**Implementation**
Added README content with demo code lines

**Screenshot**
(see the README)

**.env changes**
none

**Additional requirements**
none

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +131,35 @@ A list of some interesting expansions for the project:
...
+_9 images @ 1fps, images sorted by colour_
+
+![Alt text](./docs/gif_feature/sort_colour.gif?raw=true "9 images @ 1fps, images sorted by colour")
+
+```
+ffmpeg -y -r 1 -i ./sort_colour/%03d.jpg -r 10 -vf scale=640:480 ./sort_colour/output.gif
+```
+---
+
+**Note** : the fps could be higher, however it does create a very rapid gif with lots of colour changes - please avoid giving yourself a headache or any epilepsy users that may be in you vicinty*
```
Comment (Lily2point0) [inline README.md]: "and mind epileptic users that may be in your vicinity"

File: README.md [hunk 2]
```diff
@@ -131,6 +131,35 @@ A list of some interesting expansions for the project:
 + Look into setting cache expiry
 
 
+### Gif generation
+
+I wondered what an animated series of Barcode images would look like.
+So before building a full animation request feature I used [ffmpeg] locally to generate some example gifs.
```
Comment (Lily2point0) [inline README.md]: Add link to ffmpeg doc

File: README.md (hunk 1 above)
Comment (Lily2point0) [inline README.md]: spell out FPS at least once.

Comment (Sqyphen) [reply README.md]: 👌🏼

File: README.md (hunk 2 above)
Comment (Sqyphen) [reply README.md]: 👌🏼

File: README.md (hunk 1 above)
Comment (Sqyphen) [reply README.md]: 👌🏼

**Pull request merged**